core/amber/src/main/scala/edu/uci/ics/texera/workflow/operators/source/scan/text/TextInputSourceOpDesc.scala

```diff
@@ -16,72 +12,24 @@ import edu.uci.ics.texera.workflow.common.metadata.{
 import edu.uci.ics.texera.workflow.common.operators.source.SourceOperatorDescriptor
 import edu.uci.ics.texera.workflow.common.tuple.schema.{Attribute, OperatorSchemaInfo, Schema}
 
-import java.util.Collections.singletonList
-import javax.validation.constraints.Size
-import scala.collection.JavaConverters.asScalaBuffer
-
 class TextInputSourceOpDesc extends SourceOperatorDescriptor with TextSourceOpDesc {
   @JsonProperty(required = true)
-  @JsonSchemaTitle("Text Input")
-  @JsonSchemaDescription("Max 1024 Characters, e.g: \"line1\\nline2\"")
+  @JsonSchemaTitle("Text")
   @JsonSchemaInject(json = UIWidget.UIWidgetTextArea)
-  @Size(max = 1024)
   var textInput: String = _
 
-  // indicates the AttributeType of output tuple(s) - supports all AttributeType except ANY and BINARY
```

Comment:
why are we changing this operator? it seems to have nothing to do with the PR.

Reply:
It has shared methods with FileScan Op.